homeassistant/components/sleepiq/select.py

```diff
@@ -59,3 +71,45 @@ async def async_select_option(self, option: str) -> None:
         await self.preset.set_preset(option)
         self._attr_current_option = option
         self.async_write_ha_state()
+
+
+class SleepIQFootWarmingTempSelectEntity(
+    SleepIQSleeperEntity[SleepIQDataUpdateCoordinator], SelectEntity
+):
+    """Representation of a SleepIQ foot warming temperature select entity."""
+
+    _attr_icon = "mdi:heat-wave"
+    _attr_options = [e.name.title() for e in FootWarmingTemps]
+
+    def __init__(
+        self,
+        coordinator: SleepIQDataUpdateCoordinator,
+        bed: SleepIQBed,
+        foot_warmer: SleepIQFootWarmer,
+    ) -> None:
+        """Initialize the select entity."""
+        self.foot_warmer = foot_warmer
+        sleeper = sleeper_for_side(bed, foot_warmer.side)
+        super().__init__(coordinator, bed, sleeper, FOOT_WARMER)
+        self._async_update_attrs()
+
+    @callback
+    def _async_update_attrs(self) -> None:
+        """Update entity attributes."""
+        self._attr_current_option = FootWarmingTemps(
+            self.foot_warmer.temperature
+        ).name.title()
```

Comment:
I do still have one remark. We should use prefer to use `snake_case` for options. This way you can add the possible values to the translations file and have the values be translated. I'd rather fix it now than having to create a breaking change for everyone in a later release.